quad_utils/include/quad_utils/quad_kd.h

```diff
@@ -293,8 +404,42 @@ class QuadKD {
   std::vector<unsigned int> body_id_list_;
 
   std::vector<int> leg_idx_list_;
+
+  /// Abad max joint torque
+  const double abad_tau_max_ = 21;
```

Comment:
Do these need to be parameters?

Reply:
Likely but that would be best as part of the multi-robot PR.